app/src/main/res/layout/song_detail_dialog.xml

```diff
@@ -11,20 +11,28 @@
         android:id="@+id/imgViewSongPicture"
         android:layout_width="match_parent"
         android:layout_height="100dp"
+        android:background="@color/black"
         android:contentDescription="@string/song_picture"
         tools:srcCompat="@tools:sample/avatars" />
 
     <LinearLayout
         android:layout_width="match_parent"
         android:layout_height="match_parent"
-        android:background="@color/UC_red">
+        android:background="#2196F3"
+        android:orientation="vertical">
 
         <TextView
             android:id="@+id/txtViewSongDetails"
             android:layout_width="351dp"
             android:layout_height="100dp"
             android:text="@string/song_name" />
 
+        <TextView
+            android:id="@+id/txtView"
+            android:layout_width="351dp"
+            android:layout_height="100dp"
+            android:text="@string/artist_name" />
+
```

Comment:
1) Is this used anywhere?  Only complete features, which meet the team's Definition of Done, should be merged to master.  If this feature is not complete, I would not merge this pull request.
2) Avoid having an id of txtView.  That does not describe this widget.  Give it a descriptive name, so that you'll recognize it on the source code side.